steps/steps.py

```diff
@@ -33,3 +33,12 @@ def then_i_should_be_able_to_list_networks(context):
         assert networks, "Failed to list networks. No networks found."
         for net in networks:
             print(f"- {net['name']} ({net['id']})")
+
+    @then('I should be able to create a network with name {networkName}')
+    def then_i_should_be_able_to_create_a_network(context, networkName):
+        network = context.client.network.find_network(name_or_id=networkName)
```

Comment:
add try/except part to check if the network exists. If a network exists, the test  result should still fail but we have to get a piece of information that